GooglePlayInstant/Editor/AssetBundleVerifierWindow.cs

```diff
@@ -86,47 +93,56 @@ private void GetAssetBundleInfoFromDownload()
                 // all objects that were loaded from this bundle.
                 bundle.Unload(true);
             }
-
-            // Turn request to null to be prepared for next call
-            _webRequest.Dispose();
         }
 
         private double ConvertBytesToMegabytes(ulong bytes)
         {
             return bytes / 1024f / 1024f;
         }
 
-        //TODO: fix malformed url behavior
         private void Update()
         {
+            if (_webRequest == null || !_webRequest.isDone)
```

Comment:
Nit: we've been wrapping one line blocks in {} even though it's not required by the language

Reply:
Resolved.